media_types/deps.ts

```diff
@@ -0,0 +1,15 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+
+export { extname } from "../path/index.ts";
+
+interface DB {
+  [mediaType: string]: {
+    source?: string;
+    compressible?: boolean;
+    charset?: string;
+    extensions?: string[];
+  };
+}
+
+import * as _db from "https://raw.githubusercontent.com/jshttp/mime-db/v1.37.0/db.json";
```

Comment:
I think we should not import resources other than `deno_std`

Reply:
I disagree in this case.  We have said "limit" external dependencies, but this is a very specific external resource that would not make sense to "copy" into `std`.

Reply:
Hmm.. I agree that this is a bit different than the typical import. 

That said, I think until we have proper subresource integrity checks, we should be extra careful about external imports here, so that people can at least rely that deno_std comes from one source. (It's difficult but not impossible to imagine that db.json could updated in a nefarious way.) Please copy the file here, maybe using the version number `db_v1.37.0.json`